spec/requests/api/v1/course_spec.rb

```diff
@@ -201,7 +243,21 @@ def get_course_codes_from_body(body)
             headers: { 'HTTP_AUTHORIZATION' => credentials },
             params: { changed_since: provided_timestamp }
 
-        url = url_for(params: {
+        url = url_for(recruitment_year: 2019, params: {
+                        changed_since: provided_timestamp,
+                        per_page: 100
+                      })
+        expect(response.headers["Link"]).to match "#{url}; rel=\"next\""
+      end
+
+      it 'includes correct next link when there is an empty set' do
+        provided_timestamp = 5.seconds.ago.utc.iso8601
+
+        get '/api/v1/2020/courses',
+            headers: { 'HTTP_AUTHORIZATION' => credentials },
+            params: { changed_since: provided_timestamp }
+
+        url = url_for(recruitment_year: 2020, params: {
```

Comment:
Oh, one more nice-to-have, how about making this the full explicit URL? So it's clear exactly what this should look like.

Reply:
Have experimented. Can't get it work due to string matching not working. Asked Sau. Both couldn't figure it out. Don't want to waste more time on a nice to have.